### ISSUE TITLE
encoder training

### ISSUE THREAD

**Issue by fishman2008** [eladrich/pixel2style2pixel#201]
Hi, thanks for your excellent codebase. I am interested to train an encoder (self.encoder) and then load the weights similar as "pretrained_models/model_ir_se50.pth". It appears that the ID loss uses the same set of weights,         self.facenet.load_state_dict(torch.load(model_paths['ir_se50'])). Do they use the same set of weights? How should I train just the encoder part only? Thanks for the clarification.

**Comment by yuval-alaluf** [comment]
The encoder is indeed initialized with the same weights as used in the ID loss. However, during training the `self.facenet` you see in the `IDLoss` class remains fixed while the encoder is trained. Therefore, no alteration is needed for the code to train just the encoder network.
